src/gui/presets.cpp

```diff
@@ -931,6 +986,95 @@ void FurnaceGUI::initSystemPresets() {
       0
     }
   ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "Capcom CPS-1", { 
+      DIV_SYSTEM_YM2151, 64, 0, 2,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      0
+    }
+  ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "Jaleco Mega System 1", {
+      DIV_SYSTEM_YM2151, 64, 0, 2,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      0
+    }
+  ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "NMK Arcade", {
```

Comment:
2 MSM6295s (Both runs on 4MHz, with Bankswitching) With or Without YM2203; This configuration mostly uses at "nmk16.cpp"(from MAME source) hardware, Arcade Power instinct series(First of series uses derivative of "nmk16.cpp" hardware and same sound system, Second and Gaiden uses same sound chip and NMK112 but just different clock(4MHz vs 1.5MHz for YM2203, 3MHz vs 4MHz for MSM6295s), both runs at "cave.cpp" hardware rather than "nmk16.cpp" derivative).

Reply:
No, NMK Arcade hardware...

Reply:
> No, NMK Arcade hardware...

Actually, nmk16 seems like "NMK 16 Bit Arcade hardware".

Reply:
ok, done.